src/Agent.Service/Windows/Program.cs

```diff
@@ -12,33 +12,18 @@ static class Program
         /// </summary>
         static int Main(String[] args)
         {
-            if (args != null && args.Length == 1 && args[0].Equals("init", StringComparison.InvariantCultureIgnoreCase))
+            if (args != null && args.Length == 1)
             {
-                // TODO: LOC all strings.
-                if (!EventLog.Exists("Application"))
+                if(args[0].Equals("init", StringComparison.InvariantCultureIgnoreCase))
```

Comment:
>if(args[0].Equals("init", StringComparison.InvariantCultureIgnoreCase)) [](start = 16, length = 71)

just as a guideline can we use string.Equals(string1, string2, OrdinalIgnoreCase)